stock_view_adj_hls/static/description/index.html

```diff
@@ -381,10 +386,10 @@ <h1 class="title">Stock View Adj HLS</h1>
 </div>
 <div class="section" id="bug-tracker">
 <h1><a class="toc-backref" href="#id1">Bug Tracker</a></h1>
-<p>Bugs are tracked on <a class="reference external" href="https://github.com/qrtl/hls-custom/issues">GitHub Issues</a>.
+<p>Bugs are tracked on <a class="reference external" href="https://github.com/OCA/hls-custom/issues">GitHub Issues</a>.
```

Comment:
Why is the organization not updated here?  The same applies to the other changed parts in this file.

Reply:
Sorry, I misunderstand how to generate.
Now re-generate index.html, confirmed changed